ci/jenkins/jobs/projects.yaml

```diff
@@ -548,7 +548,7 @@
           wrappers: []
           publishers: []
       - '{name}-{test_name}-for-pull-request':
-          test_name: all-features-conformance
+          test_name: all-conformance
```

Comment:
Better to add a comment here to explain why the job name is different to trigger phrase.

Because the CI code encodes job name in CAPI cluster name, the cluster name and machine name become too long. So we use a shorter name here as a workaround. The trigger phrase will be different to Job name maybe Quan can comment if this is OK.

Reply:
Updated, thanks.

Reply:
It may sound confusing if we have both all-conformance and whole-conformance. When I looked at the build list today, I thought all-conformance was renamed from whole-conformance. Could we shorten the suffix "for-pull-request" instead? e.g. "for-pr". Then the trigger phrase will still be consistent with the job name.

Reply:
Should we rename all job names to "for-pr" or only change "all-conformance" job name?

Reply:
Either works for me. Renaming all jobs to be consistent sounds better to me if it is not a big effort.

Reply:
Thanks, maybe I should test "all-features-conformance-for-pr" firstly to see whether it works well.

Reply:
"all-features-conformance-for-pr" passed, I tend to only change this job name because of the current CI migration status. @tnqn